.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Run E2E Playwright tests
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  schedule:
```

Comment:
Drive-by comment: why do we need to run on schedule if it runs on PRs and merges anyway?